File: BI-TJV/semestral/rest-server/src/main/java/wrzecond/entity/TjvEmployee.java
package wrzecond.entity;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.SequenceGenerator;

@Entity
@EqualsAndHashCode(callSuper = true)
@AllArgsConstructor
@NoArgsConstructor
public class TjvEmployee extends TjvEntity {

    @Column(nullable = false)
    @Getter @Setter private String username;

    @Column(nullable = false)
    @Getter @Setter private String password;

    @Column(nullable = false)
    @Getter @Setter private String firstName;

    @Column(nullable = false)
    @Getter @Setter private String lastName;

    @Column(nullable = false)
    @Getter @Setter private Boolean admin;

}

File: BI-KOT/semestral/rest-server/src/main/kotlin/wrzecond/entity/TjvEmployee.kt
package wrzecond.entity

import javax.persistence.Column
import javax.persistence.Entity

@Entity
data class TjvEmployee (
    @Column(nullable = false) val username: String,
    @Column(nullable = false) val password: String,
    @Column(nullable = false) val firstName: String,
    @Column(nullable = false) val lastName: String,
    @Column(nullable = false) val admin: Boolean,
    override val id: Int = 0
) : TjvEntity()
